src/Plugin/AccessControlHierarchy/Taxonomy.php

```diff
@@ -153,7 +156,8 @@ protected function buildTree($id, array $data, array &$tree) {
         'parents' => $this->convertParents($term, $id),
         'weight' => $term->weight,
         'description' => $term->description__value,
-        'path' => Url::fromUri('entity:taxonomy_term/' . $term->tid)->toString(),
+        'path' => Url::fromUri('entity:taxonomy_term/' . $term->tid)
+          ->toString(),
```

Comment:
all of these formatting changes add noise, can you revert them?

Reply:
Done, and apologies for this. I ran phpcs/phpcbf :-)